build-tokens.js

```diff
@@ -0,0 +1,78 @@
+const StyleDictionary = require('style-dictionary');
+
+const styleDictionaryConfig = {
+  log: 'warn',
+  source: ['src/tokens/**/*.+(js|json)'],
+  platforms: {
+    scss: {
+      transformGroup: 'custom/transform-group/css',
+      buildPath: 'src/compiled/scss/',
```

Comment:
I don't care about the change from "generated" to "compiled," but why remove tokens from the path?

Reply:
Sure, I can add tokens back to the path